Rbt.roc

```diff
@@ -25,13 +25,21 @@ exec : Tool, List Str -> Command
 exec = \execTool, args ->
     Command { tool: execTool, args }
 
-Job : [Job { command : Command, inputFiles : List Str, outputs : List Str }]
+InputPath : [Path Str]
```

Comment:
it could possibly be a `Str` for now. Maybe the name is not so good here—this is where the mapping from external to workspace files will live too. `Path` is just the one that says "use the same name in the source and destination." Maybe `InputMapping` would be a better name, and would look like: `InputMapping : { from : Str, to : Str }`. Requires two strings to be allocated in the common case, but that's probably fine.

Reply:
for now, I think I'll probably just inline it. We don't need it yet, and there's no need to pay for the complexity before it's needed.